docs/content/manual/manual.yml

```diff
@@ -2277,9 +2277,10 @@ sections:
   - title: Regular expressions (PCRE)
     body: |
 
-      jq uses the Oniguruma regular expression library, as do php,
-      ruby, TextMate, Sublime Text, etc, so the description here
-      will focus on jq specifics.
+      jq uses the
+      [Oniguruma regular expression library](https://github.com/kkos/oniguruma/blob/master/doc/RE)
+      , as do php, ruby, TextMate, Sublime Text, etc, so the
```

Comment:
It's preferable to place the comma at the end of the previous line to omit spaces before it.

Reply:
Not quite sure how that happened - amended